server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationIT.java

```diff
@@ -194,6 +197,48 @@ public void testCancelPrimaryAllocation() throws Exception {
         assertSegmentStats(REPLICA_COUNT);
     }
 
+    /**
+     * This test adds a new replica shard to an existing cluster which already has few docs inserted before adding replica.
+     * We don't perform any refresh on index and assert new replica shard on doc hit count.
+     * This test makes sure that when a new replica is added to an existing cluster it gets all latest segments from primary even without a refresh.
+     */
+    public void testAddNewReplica() throws Exception {
+        logger.info("--> starting [node1] ...");
+        final String node_1 = internalCluster().startNode();
```

Comment:
nit: Usually I find it better to call nodes by their role. This makes it easier to understand when we perform any node specific actions (e.g. restart(primary), stop (replica) etc). Otherwise, we need to look back when node_i was created and its role.

Reply:
Sure, makes sense. I will rename both nodes accordingly